res/css/photon/message-bar.css

```diff
@@ -26,9 +26,13 @@
   line-height: 1.4;
 }
 
+.multi-line {
+  width: 300px;
+}
+
 .photon-message-bar-action-button {
   flex: none;
-  margin: 0 8px;
+  margin: 2px 0;
```

Comment:
I don't think we should change this one. Is there a specific reason to do this?

Reply:
I was trying to get the call-to-action-buttons to align to the start of the message @canova